sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/CreateChatThreadOptions.java

```diff
@@ -21,9 +23,9 @@
      * Members to be added to the chat thread.
      */
     @JsonProperty(value = "participants", required = true)
```

Comment:
I see that required = true was removed from the autogenerated model, we can remove it here too